### PR TITLE
Circleci debug macos

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ coverage:
       default:
         enabled: no
 comment:
-  layout: "reach, diff, flags, files, footer"
+  layout: "reach, diff, flags"
```
Comment (axic) [inline codecov.yml]: This codecov change seems unrelated? Perhaps you want to open a new PR.

**Comment by chfast** [comment]
I think that's not for merge. I wanted to debug weird failure on macOS.

File: codecov.yml (hunk 1 above)
Comment (chfast) [reply codecov.yml]: This is done.
